storage.tf

```diff
@@ -184,7 +223,7 @@ resource "aws_cloudwatch_metric_alarm" "cpu_utilization_too_high" {
   ok_actions          = var.postgresql_ok_actions
 
   dimensions = {
-    DBInstanceIdentifier = length(var.postgresql_source_snapshot_identifier) == 0 ? aws_db_instance.blank-database[0].id : aws_db_instance.from-snapshot[0].id
+    DBInstanceIdentifier = length(var.postgresql_source_snapshot_identifier) == 0 ? (length(var.postgresql_replicate_source_db) == 0 ? aws_db_instance.blank-database[0].id : aws_db_instance.replica-database[0].id) : aws_db_instance.from-snapshot[0].id
```

Comment:
should this be a nested conditional or &&?

Reply:
updated to use &&